pkg/controllers/policycache/controller.go

```diff
@@ -99,7 +99,12 @@ func (c *controller) reconcile(ctx context.Context, logger logr.Logger, key, nam
 		}
 		return err
 	}
-	return c.cache.Set(key, policy, c.client.Discovery())
+	if policy.AdmissionProcessingEnabled() {
+		return c.cache.Set(key, policy, c.client.Discovery())
```

Comment:
We need to cache the generate and mutateExisitng policies, will this break it?

Reply:
I don't think so, it's not allowed to disable admission processing for those policies.

Reply:
There's a validation check added to disallow disabling admission for mutateExisting, generate, and imageVerify(will get changed shortly) and the default is enabled.